config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -445,6 +445,51 @@ presubmits:
             privileged: true
 
 periodics:
+- interval: 168h # one week
```

Comment:
Can we add this to the existing alpha features job? https://github.com/kubernetes/test-infra/blob/72c610fe7e55ad8402966a161e8b66b41eb95912/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L340

You may also need to add the "ginkgo-parallel=1" flag since the test is disruptive.

Reply:
@msau42 This job "pull-kubernetes-e2e-gce-alpha-features" skips all sig-storage tests.  
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/111373/pull-kubernetes-e2e-gce-alpha-features/1572235414100512768/

The non-graceful node shutdown test is added under e2e/storage so it is also skipped:
```
Kubernetes e2e suite: [It] [sig-storage] [Feature:NodeOutOfServiceVolumeDetach] [Disruptive] [LinuxOnly] NonGracefulNodeShutdown [NonGracefulNodeShutdown] pod that uses a persistent volume via gce pd driver should get immediately rescheduled to a different node after non graceful node shutdown

Reason: skipped
```

Reply:
I don't see anything in the skip setting that would cause that test to be skipped:
```
---test_args=--ginkgo.focus=\[Feature:(GRPCContainerProbe|InPlacePodVerticalScaling|ProbeTerminationGracePeriod|APIServerTracing|StorageVersionAPI|PodPreset|StatefulSetMinReadySeconds|CustomResourceValidationExpressions|ProxyTerminatingEndpoints)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0 
```

With the alpha-features test suite, you need an explicit focus to opt-in the test case.

Reply:
Ok, we'll remove this job and just add the new test to the "pull-kubernetes-e2e-gce-alpha-features" job.

Reply:
@xing-yang @msau42 -- Opened a new PR https://github.com/kubernetes/test-infra/pull/27600 
Will close this once that gets merged. PTAL